data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3042,6 +3042,15 @@ spec:
                 description: GCP is the configuration used when installing on Google
                   Cloud Platform.
                 properties:
+                  clusterHostedDNS:
```

Comment:
I see the user facing install-config has also changed to `clusterHostedDNS`. IMHO, that name makes sense within the Infra CR, to represent the state of the cluster. User facing config should still indicate their behavior i.e they provide the 'userConfiguedDNS`/`userProvidedDNS`/userProvisionedDNS`.